alchemiscale/storage/statestore.py

```diff
@@ -2229,7 +2238,9 @@ def set_task_priority(
             RETURN scoped_key, t
             """
             res = tx.run(
-                q, scoped_keys=[str(t) for t in tasks], priority=priority
+                q,
+                scoped_keys=[str(t) for t in tasks if t is not None],
```

Comment:
@LilDojd did you notice a case where inputs to this method may include `None`?

Reply:
Anyway, there would be no `None`s on the server, since

```python
asc.set_tasks_priority(tasks + [None], 1)
```

Would just throw 
`AttributeError: 'NoneType' object has no attribute 'dict'` client-side

https://github.com/OpenFreeEnergy/alchemiscale/blob/4926573bf3ec29bc4e213f7586e9b7af4ca326cf/alchemiscale/interface/client.py#L1274-L1277

Reply:
Ah cool, thank you for giving your rationale @LilDojd! I see no harm in leaving this as you have done it here, as we have in `action_tasks`. Up to you if you'd like to leave it in.

Reply:
I have decided to remove `None` checks for cleanliness. Will save a few clock cycles :)